backends/ebpf/ebpfProgram.cpp

```diff
@@ -100,12 +93,19 @@ void EBPFProgram::emitC(CodeBuilder* builder, cstring header) {
     emitPipeline(builder);
 
     builder->emitIndent();
-    builder->append(endLabel);
-    builder->appendLine(":");
+    builder->appendFormat("%s:\n", endLabel);
     builder->emitIndent();
-    builder->append("return ");
-    builder->append(control->accept->name.name);
-    builder->appendLine(";");
+    builder->appendFormat("if (%s)\n", control->accept->name.name);
+    builder->increaseIndent();
+    builder->emitIndent();
+    builder->appendFormat("return %s;\n", builder->target->forwardReturnCode());
+    builder->decreaseIndent();
+    builder->emitIndent();
+    builder->appendLine("else");
+    builder->increaseIndent();
+    builder->emitIndent();
+    builder->appendFormat("return %s;\n", builder->target->dropReturnCode());
+    builder->decreaseIndent();
```

Comment:
I do not particularly like this change, but it was necessary to get the kernel target to work. This section basically checks if "pass" is true. If that is the case, we emit the forward code, otherwise we drop the packet. The true/false return code conflicts with the codes, tc expects, which is why this change was necessary. What is the purpose of "pass" versus the target codes?